README.md

```diff
@@ -15,51 +15,57 @@
 <h3 align="center">
   <a href="https://docs.aklivity.io/zilla/"><b>Documentation</b></a> &bull;
   <a href="https://docs.aklivity.io/zilla/latest/tutorials/quickstart/kafka-proxies.html"><b>Quickstart</b></a> &bull;
-  <a href="https://github.com/aklivity/zilla-examples"><b>Examples</b></a> &bull; 
+  <a href="https://github.com/aklivity/zilla-examples"><b>Examples</b></a> &bull;
   <a href="https://www.aklivity.io/blog"><b>Blog</b></a>  
 </h3>
 
+# A multi-protocol edge and service proxy designed for event-driven architectures
 
-##  A multi-protocol edge and service proxy, designed for event-driven architectures
-Zilla abstracts Apache Kafka® for web applications, IoT clients and non-Kafka microservices. With Zilla, Kafka topics can be securely and reliably exposed via a user-defined `REST`, `Server-Sent Events (SSE)`, `MQTT`, or `gRPC` API.
+Zilla abstracts Apache Kafka® for web applications, IoT clients, and non-Kafka microservices. Zilla can securely and reliably expose Kafka topics via a user-defined `REST`, `Server-Sent Events (SSE)`, `MQTT`, and `gRPC` APIs.
 
-Zilla has no external dependencies and does not rely on the Kafka Consumer/Producer API or Kafka Connect. Instead, it natively supports the Kafka wire protocol and uses advanced protocol mediation to establish stateless API entry points into Kafka. Besides protocol meditation, Zilla addresses security enforcement, observability and connection offloading on the data path.
+Zilla has no external dependencies and does not rely on the Kafka Consumer/Producer API or Kafka Connect. Instead, it natively supports the Kafka wire protocol and uses advanced protocol mediation to establish stateless API entry points into Kafka. Zilla also addresses security enforcement, observability, and connection offloading on the data path.
```

Comment:
The number of connections from clients into Zilla does not have to equal the number of connections coming out of Zilla into Kafka , i.e. fan-in and fan-out.

Reply:
Ok that makes more sense. If we can find where this terminology is documented, I am good with it. Otherwise, we may want to figure out how to reword it. Particularly since that is a huge feature and part of the first few sentences